### PR TITLE
Add tests for .textEqual()

### PR DESCRIPTION
**Pull request by fernandopasik**
* Add 3 tests for .textEqual()
* Do coverage only for the generator file

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -120,6 +120,27 @@ describe('yeoman-assert', function () {
     });
   });
 
+  describe('.textEqual()', function () {
+    it('pass with two similar simple lines', function () {
+      assert.doesNotThrow(yoAssert.textEqual.bind(yoAssert,
+        'I have a yellow cat',
+        'I have a yellow cat'
+      ));
+    });
```
Comment (kevva) [inline test.js]: Add empty line between the tests.

Comment (fernandopasik) [inline test.js]: Done!

**Comment by kevva** [comment]
lgtm

**Comment by SBoudrias** [comment]
Thanks!

**Pull request merged**